test/ResultTest.cpp

```diff
@@ -365,7 +366,19 @@ TEST(Result, verifyApplyLimitOffsetDoesCorrectlyApplyLimitAndOffset) {
     EXPECT_EQ(totalRows, 0);
     EXPECT_TRUE(colSizes.empty());
 
-    consumeGenerator(result.idTables());
+    for (const auto& innerTable : result.idTables()) {
+      for (const auto& row : innerTable) {
+        ASSERT_EQ(row.size(), 2);
+        // Make sure we never get values that were supposed to be filtered
+        // out.
+        EXPECT_NE(row[0].getVocabIndex().get(), 0);
+        EXPECT_NE(row[0].getVocabIndex().get(), 1);
+        EXPECT_NE(row[0].getVocabIndex().get(), 4);
```

Comment:
```suggestion
     EXPECT_THAT(row[0].getVocabIndex(), Not(AnyOf(0, 1, 4));
```

Similar with the the next three lines.